llvm/test/CodeGen/AArch64/GlobalISel/ptrauth-lowering-err-debug.ll

```diff
@@ -0,0 +1,58 @@
+; REQUIRES: asserts
+; RUN: split-file %s %t && cd %t
+
+;--- MOVaddrPAC.ll
+
+; RUN: not --crash llc -debug -mtriple aarch64-elf MOVaddrPAC.ll 2>&1 | \
```

Comment:
`-debug` may be replaced with `-debug-only=aarch64-expand-hardened-pseudos` for less verbose debug output (not sure which is more canonical in tests, though).

Reply:
+1

Reply:
Thanks, fixed in 52afe428f699b1eddcd6b649058d1e21c2f21b1b